gateway/ln-gateway/src/actor.rs

```diff
@@ -96,39 +178,39 @@ impl GatewayActor {
         })
         .await;
 
+        // Create a channel that will be used to shutdown the HTLC thread
+        let (sender, receiver) = mpsc::channel::<Arc<AtomicBool>>(100);
```

Comment:
This is currently somewhat difficult because `GatewayActor` needs to be clonable. After [this refactor](https://github.com/fedimint/fedimint/issues/2192) though I think it should be possible